ee/clickhouse/views/test/test_clickhouse_retention.py

```diff
@@ -49,13 +49,16 @@ def test_can_get_retention_cohort_breakdown(self):
             ),
```

Comment:
Tedious but worth moving and converting ee/clickhouse/queries/test/test_retention.py tests here and use the new helpers so we can gradually unify the tests. (Can also do this as a follow up)

Reply:
I think work leaving this to a separate pr? 🤞